src/gluonnlp/loss/loss.py

```diff
@@ -0,0 +1,96 @@
+# coding: utf-8
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Loss functions."""
+
+__all__ = ['SoftmaxCEMaskedLoss']
+
+import numpy as np
+from mxnet.gluon.loss import SoftmaxCELoss
+
+class SoftmaxCEMaskedLoss(SoftmaxCELoss):
```

Comment:
As we move this to the main API, some things to consider (@szha):
- Should this be an alias to `SoftmaxCrossEntropyMaskedLoss` (similar to [Gluon proper](https://mxnet.incubator.apache.org/api/python/gluon/loss.html#mxnet.gluon.loss.SoftmaxCrossEntropyLoss))?
- Should this be `MaskedSoftmaxCELoss` instead (e.g., [as in AllenNLP](https://allenai.github.io/allennlp-docs/api/allennlp.nn.util.html))? More generally, how should we name Masked variants of things?

Reply:
+1 on MaskedSoftmaxCrossEntropyLoss and its alias MaskedSoftmaxCELoss